conf/global/console.conf

```diff
@@ -38,7 +38,8 @@ console: {
 	//  %I:%M:%S %p (hour:minute:second 12 hour, AM/PM format)
 	//  %H:%M:%S (hour:minute:second, 24 hour format)
 	//  %d/%b/%Y (day/Month/year)
-	// For full format information, consult the strftime() manual.
+	// For full format information, consult the strftime() manual on
+	// <https://www.php.net/manual/en/function.strftime.php>
```

Comment:
`strftime` is not a php function so you should not link to the php version of it, you can use https://man7.org/linux/man-pages/man3/strftime.3.html